### PR TITLE
Export to static JSON files on S3

### LINKED ISSUE #241: Export user's makes into JSON hosted on S3
As a part of [archiving](https://blog.webmaker.org/whats-next-for-webmaker-tools) user's makes:

For each user, create a JSON file with all of the user's makes, upload file to S3 in a user-space URL. (e.g. /archivedMakes/{user_id}.json

```JSON
{
  makes: [
    {
      email: "fake@123.com",
      url: "http://notarealurl.com/make_asdf.html",
      contentType: "application/x-applicationName",
      title: "title of make",
      contenturl: "http://realMakeURL.com/notbehindaproxy",
      locale: "en_CA",
      description: "description of make",
      thumbnail: "http://fakeimageURL.com/img123.gif",
      author: "anonymous person",
      tags: [
          "foo",
          "bar"
      ],
      remixedFrom: "OBJECT-ID-OF-ANOTHER-MAKE"
    },
    ...
  ]
  total: n
}
```

### PR DESCRIPTION
**Pull request by cadecairos**
Fixes #241 

required to run:

S3_EXPORT_BUCKET - the bucket where makes should be exported to
AWS_S3_EXPORT_ACCESS_KEY - AWS Access key with permissions to write to the bucket specified above
AWS_S3_EXPORT_SECRET_ACCESS_KEY - The secret access key that pairs with the above access key

```node scripts/export_to_s3.js```

r? @jbuck

### CODE REVIEW

File: scripts/export_to_s3.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
+var async = require('async');
+var Habitat = require('habitat');
```
Comment (jbuck) [inline scripts/export_to_s3.js]: Do you need habitat here if you're using env below?

Comment (cadecairos) [inline scripts/export_to_s3.js]: yeah we can drop that.

File: scripts/export_to_s3.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+var AWS = require('aws-sdk');
+var env = require('../lib/environment');
+var mongoose = require( '../lib/mongoose' )(connectionReady);
+var Make = require( '../lib/models/make' )( mongoose.mongoInstance() );
+
+var documentCount;
+var aggregatedMakes = {};
+var aggregatedRedactedMakes = {};
+var currentSkipValue = 0;
+var aggregationLimit = 300;
```
Comment (jbuck) [inline scripts/export_to_s3.js]: this seems pretty low; is each make actually 50KB? (16MB / 300 makes = 53.3KB/make). Or is this the only number that worked in practice?

Comment (cadecairos) [inline scripts/export_to_s3.js]: We can bump it up and see. on staging, this step takes ~15s

Comment (cadecairos) [inline scripts/export_to_s3.js]: set it to 2500, still works!

**Comment by jbuck** [comment]
@cadecairos r+

**Pull request merged**